add_to_app/README.md

```diff
@@ -62,3 +62,21 @@ iOS or Android app, the best places to go are:
 
 If you run into an issue with the sample itself, please file an issue
 in the [main Flutter repo](https://github.com/flutter/flutter/issues).
+
+## Updating Android samples and its dependencies
+
+TODO @ewindmill: Expand these instructions for those of us with little Android experience :)
+
+* Open the top level build.gradle file in android studio
+* Open “Upgrade Assistant” and click upgrade
+* If after upgrading it fails to build try upgrading to a slightly less new version
+* If you get an unknown class version exception 61, 66, etc, try setting the version of Java used by modifying JAVA_HOME
+* If it still fails to build try checking if the Flutter code referenced in the build is up-to-date and doesn't use discontinued plugins. (Common cases include “namespace”)
+* The 'project structure' -> 'modules' view can be helpful in understanding the dependency tree.
+* Once the app builds with the latest gradle/agp update any deprecated usages in app/build.gradle
+
+* compileSdkVersion -> sdkVersion
+* Update the target sdk version and read through each target sdk update
+* Export broadcast receivers
+* Update the way flutter is imported to use flutter gradle plugin
+* https://flutter.dev/go/flutter-gradle-plugin-apply
```

Comment:
when updating an app that uses a flutter module as an aar 
* in android studio update any android dependencies that are indicated as in yellow as old. 
Keep `androidx.test:runner`, `androidx.test.espresso:espresso-core`, and `androidx.test:core`,  as defined in https://github.com/flutter/packages/tree/main/packages/espresso